contract-tests/sse-contract-tests/src/entity_manager.cpp

```diff
@@ -53,7 +59,7 @@ std::optional<std::string> EntityManager::create(ConfigParams const& params) {
         return std::nullopt;
     }
 
-    client->run();
```

Comment:
Renamed `run` to `async_connect` to make it clear this is kicking off an asynchronous operation.